src/script/interpreter.cpp

```diff
@@ -535,8 +535,13 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
 
                 case OP_CAT:
                 {
+                    // OP_CAT is only available in Tapscript
+                    if (sigversion == SigVersion::BASE || sigversion == SigVersion::WITNESS_V0)
+                        return set_error(serror, SCRIPT_ERR_BAD_OPCODE);
+
```

Comment:
This blank line in "OP_CAT only available in tapscript" has trailing spaces, causing the lint check to fail

Reply:
This should be fixed in the latest commit [86ab1b1](https://github.com/bitcoin-inquisition/bitcoin/pull/39/commits/86ab1b12be36b4b411d3b0dc4409fac002a2139d)

(the linter is passing via `docker run --rm -v $(pwd):/bitcoin -it bitcoin-linter` on my local)

Reply:
Try `git log v25.0-inq..HEAD | grep commit | while read a b c; do git show $b | sed 's/.. $/& <<<< HERE/'; echo =====; done | less` and look for the HERE markers? There's trailing whitespace introduced in commits cfea5bb901d47a7b531794f064afeb9f30a861e7 and 09f975c3b03573917c92405be1688a301530032a.

Reply:
That worked! Thanks